app/controllers/users/relationships_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Users::RelationshipsController < ApplicationController
+  def create
+    current_user.active_relationships.create!(followed_id: params[:user_id])
+    redirect_to request.referer || root_path
+  end
+
+  def destroy
+    current_user.active_relationships.find_by(followed_id: params[:id]).destroy!
```

Comment:
destroy アクションのときの params[:id] は relationship の id になるはずなので、
こう消すのが正しそう。
```suggestion
    current_user.active_relationships.find(params[:id]).destroy!
```